city_scrapers/spiders/ind_indygo.py

```diff
@@ -17,79 +27,36 @@ def parse(self, response):
         Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
         needs.
         """
-        for item in response.css(".content-section:nth-child(4) ul li"):
+        meeting_section = response.css(".content-section:nth-child(4)")
+        section_title = meeting_section.css("h2::text").get()
+        meeting_year = section_title.split(" ")[0]
+        meeting_time = (
+            (response.css(".content-section:nth-child(4) p strong::text").get())
+            .split(":", 1)[1]
+            .split("EST")[0]
+            .strip()
+        )
```

Comment:
I like how you're scraping the meeting time in addition to the year! Hopefully this will make the scraper more robust if the agency changes the meeting time down the road. Just for a little polish here: it looks like an extra set of brackets is enclosing the operation on the response object on line 34. The chained operations here still work but those brackets aren't needed. Can you please tweak that to this (sans my comment):

```
        meeting_time = (
            response.css(".content-section:nth-child(4) p strong::text").get() # brackets removed here
            .split(":", 1)[1]
            .split("EST")[0]
            .strip()
        )
```